### ISSUE TITLE
credentials_path variable not working

### ISSUE THREAD

**Issue by as-vspry** [terraform-google-modules/terraform-google-project-factory#671]
### TL;DR

The [readme](https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/latest#permissions) suggests that path to a service account key can be the variable `credentials_path` to control the service account used by project factory. However, this variable is not listed in the inputs on the terraform registry and doesn't pass a `terraform validate` check. 

### Expected behavior

A path to a service account key can be provided to the `credentials_path` argument, and project factory will perform scripts and actions using this service account, as per the documentation: [Specifying credentials](https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/latest#permissions).

### Observed behavior

`terraform validate result`: 
│ Error: Unsupported argument
│ 
│   on main.tf line 36, in module "project-factory":
│   36:   credentials_path     = "key-file.json"
│ 
│ An argument named "credentials_path" is not expected here.

`terraform apply result`:
│ Error: Unsupported argument
│ 
│   on main.tf line 36, in module "project-factory":
│   36:   credentials_path     = "key-file.json"
│ 
│ An argument named "credentials_path" is not expected here.

### Terraform Configuration

```hcl
terraform {
  required_providers {
    google = {
      source = "hashicorp/google"
      # version = "4.5.0"
    }
    google-beta = {
      source = "hashicorp/google-beta"
      # version = "4.5.0"
    }
  }
}

variable "project" {
  type    = string
  default = "<project-name>"
}

variable "region" {
  type    = string
  default = "<region>"
}

provider "google" {
  credentials = file("key-file.json")
  project     = var.project
  region      = var.region
}

module "project-factory" {
  source  = "terraform-google-modules/project-factory/google"
  version = "11.3.0"

  # insert the 7 required variables here
  credentials_path     = "key-file.json"
  name                 = "terraform-test-1"
  random_project_id    = true
  org_id               = "0"
  usage_bucket_name    = "pf-test-1-usage-report-bucket"
  usage_bucket_prefix  = "pf/test/1/integration"
  billing_account      = "<BILLING-ACCOUNT-ID>"
  svpc_host_project_id = ""
}
```

### Terraform Version

```sh
Terraform v1.0.11
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v4.5.0
+ provider registry.terraform.io/hashicorp/google-beta v4.5.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/time v0.7.2

Your version of Terraform is out of date! The latest version
is 1.1.2. You can update by downloading from https://www.terraform.io/downloads.html
```

**Comment by morgante** [comment]
Since v11, you no longer need to set a `credentials_path` variable. Setting it on the provider alone is sufficient (as you have done): https://github.com/terraform-google-modules/terraform-google-project-factory/blob/e4c9b0362166ebb2ce6cab1a54d55fc3014f00f3/docs/upgrading_to_project_factory_v11.0.md#unused-variables-have-been-removed

**Comment by as-vspry** [comment]
That's even more convenient, thanks!
